src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/JournalpostService.kt

```diff
@@ -98,6 +95,23 @@ class JournalpostService(private val journalpostClient: JournalpostClient, priva
         journalpostClient.oppdaterJournalpost(oppdatertJournalpost, journalpost.journalpostId, saksbehandler)
     }
 
+    private fun oppdaterLogiskeVedlegg(journalpost: Journalpost, logiskeVedlegg: Map<String, List<LogiskVedlegg>>?) {
+        if (logiskeVedlegg == null) return
+
+        journalpost.dokumenter?.forEach { dokument ->
+            val eksisterendeLogiskeVedlegg = dokument.logiskeVedlegg ?: emptyList()
+            val logiskeVedleggForDokument = logiskeVedlegg[dokument.dokumentInfoId] ?: emptyList()
+            val harIdentiskInnhold =
+                eksisterendeLogiskeVedlegg.containsAll(logiskeVedleggForDokument) && eksisterendeLogiskeVedlegg.size == logiskeVedleggForDokument.size
```

Comment:
Burde denne sjekken snus? Eller gjøres begge sjekkene uansett om den første feiler? For sjekk på size går vell raskere?

Reply:
Har nok ikke så mye å si, men kan godt snu den.